src/lights/PointLightShadow.d.ts

```diff
@@ -0,0 +1,8 @@
+import { OrthographicCamera } from './../cameras/OrthographicCamera';
+import { LightShadow } from './LightShadow';
+
+export class PointLightShadow extends LightShadow {
+
+	camera: OrthographicCamera;
```

Comment:
This should be `PerspectiveCamera`, right?

Reply:
Yep, nice spot.